src/helpers/PointLightHelper.js

```diff
@@ -86,6 +84,14 @@ class PointLightHelper extends Mesh {
 
 	}
 
+	updateMatrixWorld( force ) {
+
+		this.update();
+
+		return super.updateMatrixWorld( force );
+
+	}
+
```

Comment:
`webgl_helpers` relied on this helper being automatically updated from `this.matrix = this.light.matrixWorld` without a call to `update`. This emulates earlier behavior.